src/games/strategy/triplea/ResourceLoader.java

```diff
@@ -98,14 +99,16 @@ public boolean match(final File f) {
         return f.exists();
       }
     });
-    if (existing.size() > 1) {
-      System.out.println("INFO: Found too many files for: " + mapName + "  found: " + existing);
-    }
+
+
     // At least one must exist
     if (existing.isEmpty()) {
       return Collections.emptyList();
     }
     final File match = existing.iterator().next();
+    if (existing.size() > 1) {
```

Comment:
@DanVanAtta you still didn't got the error.
This is occuring, even when there is only one map in one folder...
The Problem is that if `normalizedZipName` is equals `zipName` there are to *identical* entries in `existing`... e.g. `existing.size()` is greater than 1 although there is only a single Map file (written all lowercase with no spaces...)
#620 would have fixed that...